students/etwum/lesson09/mailroom_part5.py

```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/env python3
+
+# initial list of donors
+
+
+class DonorData:
+    def __init__(self,donors):
+        self.donors = donors
+
+    def donor_data(self):
+        return self.donors
+
+
+class Interaction:
+
+    def __init__(self, donor_list = None):
+        if donor_list is None:
+            donor_list = []
+        else:
+            self.donor_list = donor_list
+
+    def add_new_donor(self,new_donor):
+        # method to add a new donor
+        self.donor_list.append(new_donor)
+
+    def send_thank_you(self):
```

Comment:
this function is a bit too big, would be good to break it up into smaller functions for readability